docs/howto/openstack/neutron/new-network-control-panel.md

```diff
@@ -1,57 +1,332 @@
+    | subnetpool_id        | None                                 |
+    | tags                 |                                      |
+    | updated_at           | 2022-10-30T14:47:40Z                 |
+    +----------------------+--------------------------------------+
+    ```
+
+    You will need the subnet `id` in subsequent commands, so write it
+    down. Better yet, assign the `id` value to an environment
+    variable:
+
+    ```bash
+    SUBNET_ID="1b0822b3-62e8-4b40-92e8-8544c72d4c15"
+    ```
+
+    If you want servers connected to the `nuremberg` network to have
+    Internet access, you need a router in front of the
+    network. Following our unofficial naming convention, go ahead and
+    create a new router called `nuremberg-router`:
+
+    ```bash
+    openstack router create nuremberg-router 
+    ```
+
+    As expected, you will see lots of information regarding the new router:
+
+    ```plain
+    +-------------------------+--------------------------------------+
+    | Field                   | Value                                |
+    +-------------------------+--------------------------------------+
+    | admin_state_up          | UP                                   |
+    | availability_zone_hints |                                      |
+    | availability_zones      |                                      |
+    | created_at              | 2022-10-30T15:36:26Z                 |
+    | description             |                                      |
+    | enable_ndp_proxy        | None                                 |
+    | external_gateway_info   | null                                 |
+    | flavor_id               | None                                 |
+    | ha                      | True                                 |
+    | id                      | 566de991-fc0e-4f85-b6c4-5c87694781f7 |
+    | name                    | nuremberg-router                     |
+    | project_id              | dfc700467396428bacba4376e72cc3e9     |
+    | revision_number         | 1                                    |
+    | routes                  |                                      |
+    | status                  | ACTIVE                               |
+    | tags                    |                                      |
+    | tenant_id               | dfc700467396428bacba4376e72cc3e9     |
+    | updated_at              | 2022-10-30T15:36:26Z                 |
+    +-------------------------+--------------------------------------+
+    ```
+
+    You will soon need the router `id` so assign the `id` value to an
+    environment variable:
+
+    ```bash
+    ROUTER_ID="566de991-fc0e-4f85-b6c4-5c87694781f7"
+    ```
+
+    You want the `nuremberg_router` connected to the external
+    network. To realize this connection, you need the `id` of said
+    network. Go ahead and list all available networks...
+
+    ```bash
+    openstack network list -f json
```

Comment:
Please don't use `-f json`. When we create objects in OpenStack, the output we get is tabular (that is, it is equivalent to the default format, i.e. `-f table`). Instructing users to then read back the object data in a different format is a [POLA](https://en.wikipedia.org/wiki/Principle_of_least_astonishment) violation.

Reply:
Aha, I never heard of POLA! The thing is that tabular output --at least on my terminal-- seems ugly -- and what I really want to say is "broken". That's why I went for JSON. Do you think we could ignore POLA, at least for this post?

Reply:
You can never "ignore" POLA.

And, we use tabular output everywhere else. If you want to recommend switching to JSON in this case, you'd either have to change *all* `openstack <object> [show|list]` examples throughout,  you'd at the very least have to explain (to readers in the text, not to me in the review thread) why you do so. "The other format is ugly" is not a sufficient explanation, really. If the other format is *broken,* then we shouldn't use it anywhere, but I don't exactly see the brokenness. Can you elaborate?

Reply:
Well, this is how tabular output looks on my terminal (and that's why I went for JSON).

<img width="990" alt="Screen Shot 2022-10-31 at 14 31 51" src="https://user-images.githubusercontent.com/20104288/199009441-28ba0b0c-0545-4cfa-aa84-b42e581484d7.png">

Reply:
In that case generate the terminal output with ``openstack network list --max-width 100``.

Reply:
See also: #46 (I guess this is quite a helpful tweak to suggest to users).

Reply:
Agreed, setting CLIFF_FIT_WIDTH solves any issues regarding the looks of tabular output. I am going ahead and making all the necessary edits.